frontend/src/metabase/components/Select.jsx

```diff
@@ -4,11 +4,22 @@ import ColumnarSelector from "metabase/components/ColumnarSelector.jsx";
 import Icon from "metabase/components/Icon.jsx";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
 
+import cx from "classnames";
+
 export default class Select extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            value: null
+        };
+    }
+
     static propTypes = {
         value: PropTypes.any,
+        changeOnSelect: PropTypes.bool,
         options: PropTypes.array.isRequired,
         placeholder: PropTypes.string,
+        triggerClasses: PropTypes.string,
```

Comment:
You should use `className` if classes end up on the root element of the component, as I believe they do in this case. (`Triggerable` is sort of a special case)